koji/foreman-client-mash-split.py

```diff
@@ -318,7 +318,7 @@ def main():
     else:
         git_tag = "rpm/{}".format(version)
 
-        if version in ('1.22', '1.23', '1.24'):
+        if version in ('1.23', '1.24', '2.0'):
```

Comment:
2.0 does have el8 support. We also haven't dropped the 1.22 pipelines yet so until then I'd keep it in.
```suggestion
        if version in ('1.22', '1.23', '1.24'):
```